packages/cozy-harvest-lib/src/components/KonnectorConfiguration/DataTab/index.jsx

```diff
@@ -70,6 +71,9 @@ export const DataTab = ({ konnector, trigger, client, flow }) => {
         {appLinks.map(({ slug, ...otherProps }) => (
           <AppLinkCard key={slug} slug={slug} {...otherProps} />
         ))}
+        {konnector.vendor_link && (
```

Comment:
Does this handle the trainline case where vendor_link can be an array ?

Reply:
Actually, the trainline case is not an array, it is just an invalid url (without, https://)